startvscode.cmd

```diff
@@ -0,0 +1,32 @@
+@ECHO OFF
```

Comment:
Can you implement a Shell-script based variant of this a la `startvscode.sh` for users on mac/Linux?

Let me know if you need help with this.

Reply:
Ummm, need to check. I will try to do it this weekend!

Reply:
Hi @captainsafia, Sorry, am not much familiar with MacOS and it's shell working! 

If you managed to get time to do that and wanted to get that done in this PR only, I think you can push commit to this PR as well! Otherwise better to have a follow up PR after this! 😇

Reply:
Got it -- we can address this in a follow-up PR.

Reply:
Very thanks @captainsafia! 😇